example.typ

```diff
@@ -55,5 +55,4 @@
 #cvpublications(cvdata)
 #cvskills(cvdata)
 #cvreferences(cvdata)
-
-#endnote
+#endnote()
```

Comment:
Also I updated `endnote` to be a "function". It might break other's cv if they don't update it to be a function tho. Let me know if it is acceptable or not.